File: examples/file/list/node/listFiles.js
const { FileService } = require("m3o/file");

// List files by their project and optionally a path.
async function listFiles() {
  let fileService = new FileService(process.env.MICRO_API_TOKEN);
  let rsp = await fileService.list({
    project: "examples",
  });
  console.log(rsp);
}

listFiles();

File: examples/go/file/list/listFiles.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/file"
)

// List files by their project and optionally a path.
func ListFiles() {
	fileService := file.NewFileService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := fileService.List(&file.ListRequest{
		Project: "examples",
	})
	fmt.Println(rsp, err)
}
